src/App.tsx

```diff
@@ -1,12 +1,60 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import React, { useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import cn from 'classnames';
+import { Footer } from './components/Footer';
+import { ErrorMessages } from './types/ErrorMessages';
+import { Filter } from './types/Filter';
+import { getFilteredList } from './utils/getFilteredList';
+
+const errorMessages: ErrorMessages = {
+  load: 'Unable to load todos',
+  title: 'Title should not be empty',
+  add: 'Unable to delete a todo',
+  delete: 'Unable to delete a todo',
+  update: 'Unable to update a todo',
+};
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todosList, setTodosList] = useState<Todo[] | null>(null);
+  const [filter, setFilter] = useState<Filter>('All');
+
+  const [error, setError] = useState<keyof ErrorMessages | null>(null);
+
+  useEffect(() => {
+    getTodos()
+      .then(data => setTodosList(data))
+      .catch(() => {
+        setError('load');
```

Comment:
but you create object for this purpose, let's use it everywhere, your code shouldn't rely of magic string
```suggestion
        setError(errorMessages.load);
```